wire/modules/Process/ProcessPageEditImageSelect/ProcessPageEditImageSelect.module

```diff
@@ -289,6 +290,11 @@ class ProcessPageEditImageSelect extends Process implements ConfigurableModule {
 		}
 		
 		if(!$id) throw new WireException("No page specified");
+		if($this->sourcePage) {
+			$this->page = $this->pages->get($this->sourcePage);
```

Comment:
Not particularly familiar with how this module works, but should this be findOne instead of get? Just wondering what would happen if current user doesn't have access to the page, since get bypasses access/status checks :)